Code/Framework/AzToolsFramework/Platform/Mac/platform_mac_files.cmake

```diff
@@ -7,6 +7,8 @@
 #
 
 set(FILES
+    AzToolsFramework_Traits_Mac.h
+    AzToolsFramework_Traits_Platform.h
     AzToolsFramework/API/EditorAssetSystemAPI_Mac.cpp
     AzToolsFramework/API/PythonLoader_Mac.cpp
```

Comment:
The PythonLoader_Mac.cpp file has been removed. So this entry needs to be removed or MacOS will fail to CMake configure.

Reply:
Very strange, my local file was updated, but git didn't pick up the diff.

Reply:
Git doesn't seem to want to pick up my change
```
diff --git a/Code/Framework/AzToolsFramework/Platform/Mac/platform_mac_files.cmake b/Code/Framework/AzToolsFramework/Platform/Mac/platform_mac_files.cmake
index 1c304c14ea..92ea56f946 100644
--- a/Code/Framework/AzToolsFramework/Platform/Mac/platform_mac_files.cmake
+++ b/Code/Framework/AzToolsFramework/Platform/Mac/platform_mac_files.cmake
@@ -9,6 +9,6 @@
 set(FILES
     AzToolsFramework_Traits_Mac.h
     AzToolsFramework_Traits_Platform.h
+    ../Common/UnixLike/AzToolsFramework/API/PythonLoader_UnixLike.cpp
     AzToolsFramework/API/EditorAssetSystemAPI_Mac.cpp
-    AzToolsFramework/API/PythonLoader_Mac.cpp
 )
```